### PR TITLE
Remove space before nolint comment to make it consistent across the project

### PR DESCRIPTION
**Pull request by surajssd**
This commit replaces `// nolint` with `//nolint`, to make it consistent
across the project.

### CODE REVIEW

**Review by invidian — CHANGES_REQUESTED** [review]
@surajssd I believe this should be other way around `// nolint` -> `//nolint`, as this is how upstream has it documented: https://golangci-lint.run/usage/false-positives/.

File: cli/cmd/completion.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ const bashCompDesc = `  Generate the completion code for lokoctl for the Bash sh
 
 const bashExample = `  # If running Bash 3.2 that is included with macOS, install Bash completion using Homebrew.
   brew install bash-completion
-
```
Comment (invidian) [inline cli/cmd/completion.go:46]: Seems unrelated.

**Review by invidian — APPROVED** [review]

**Review by knrt10 — APPROVED** [review]

**Comment by knrt10** [comment]
I don't think today Packet CI will let us merge

**Comment by invidian** [comment]
>I don't think today Packet CI will let us merge

And yet, it did :)
